package.json

```diff
@@ -70,7 +70,7 @@
   },
   "scripts": {
     "start": "nf start -p 3000 && electron-start",
-    "electron": "yarn build-electron && electron .",
+    "electron-start": "yarn build-electron && electron .",
```

Comment:
I'm not really using the start command anymore because with it is not possible to debug electron. So is better to open two terminal windows and run yarn react-start on one and yarn electron on the other. Then you can read console.log that you put on electron. So maybe I'll just get rid of the start command.

Reply:
Alright, I've just been doing start in vscode but I'll do that instead.

Reply:
I changed my mind because I was able to fix the start command. So it is working as before. Check on the main branch.